browser/mapping.go

```diff
@@ -149,7 +149,12 @@ func mapElementHandle(ctx context.Context, vu k6modules.VU, eh api.ElementHandle
 		"objectID":       eh.ObjectID,
 		"boundingBox":    eh.BoundingBox,
 		"check":          eh.Check,
-		"click":          eh.Click,
+		"click": func(opts goja.Value) *goja.Promise {
+			return k6ext.Promise(ctx, func() (any, error) {
+				err := eh.Click(opts)
+				return nil, fmt.Errorf("%w", err)
```

Comment:
Forgot: `//nolint:wrapcheck` here. Harmless, but, you know.

Reply:
Maybe make it consistent. I prefer `fmt.Errorf("%w", err)` over `//nolint:wrapcheck`. I leave it you though 👍

Reply:
We decided against that in sync back in July and disabled the wrapcheck linter. I'm confused about what changed your mind in that time frame 😄 I prefer the second, the first one is unnecessarily verbose and `%w` should not be preferred unless you want to unwrap the error. I'll go with the wrapcheck.

Reply:
I can't remember that far back :) Happy with the disabling of the wrapcheck linter. Why do we need the nolint still if the wrapcheck is disabled?